From cb49f587d493daa2d3142e69bcb2492d18aabc17 Mon Sep 17 00:00:00 2001
From: kirubagharanas <kirubagharanas@ami.com>
Date: Thu, 9 Jan 2025 00:18:32 +0530
Subject: [PATCH] Add HttpPushUriTarget and busy property under software

This change adds the HttpPushUriTargets and HttpPushUriTargetsBusy D-bus property
under xyz.openbmc_project.Software.BMC.Updater.

---
 item_updater.hpp | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/item_updater.hpp b/item_updater.hpp
index c691272..5e4c22b 100644
--- a/item_updater.hpp
+++ b/item_updater.hpp
@@ -10,6 +10,7 @@
 #include <xyz/openbmc_project/Common/FactoryReset/server.hpp>
 #include <xyz/openbmc_project/Control/FieldMode/server.hpp>
 #include <xyz/openbmc_project/Software/ApplyOptions/server.hpp>
+#include <xyz/openbmc_project/Software/FirmwareUpdateTarget/server.hpp>
 
 #include <string>
 #include <vector>
@@ -26,7 +27,8 @@ using ItemUpdaterInherit = sdbusplus::server::object_t<
     sdbusplus::server::xyz::openbmc_project::control::FieldMode,
     sdbusplus::server::xyz::openbmc_project::association::Definitions,
     sdbusplus::server::xyz::openbmc_project::collection::DeleteAll,
-    sdbusplus::xyz::openbmc_project::Software::server::ApplyOptions>;
+    sdbusplus::xyz::openbmc_project::Software::server::ApplyOptions,
+    sdbusplus::xyz::openbmc_project::Software::server::FirmwareUpdateTarget>;
 
 namespace MatchRules = sdbusplus::bus::match::rules;
 using VersionClass = phosphor::software::manager::Version;
-- 
2.34.1

